tests/exploits/cve_2024_6387_test.py

```diff
@@ -0,0 +1,60 @@
+from unittest import mock
+from unittest.mock import patch
+from agent import definitions
+from agent.exploits import cve_2024_6387
+
+
+def create_target(scheme: str, domain: str, port: int) -> definitions.Target:
+    """Helper function to create a Target instance.
+
+    Args:
+        scheme: The URL scheme.
+        domain: The domain or IP address.
+        port: The port number.
+
+    Returns:
+        A Target instance.
+    """
+    return definitions.Target(scheme, domain, port)
+
+
+@patch("agent.exploits.cve_2024_6387._get_ssh_banner")
+def test_open_ssh_exploit_when_vulnerable_report_finding(
```

Comment:
Unit test names usually follow this pattern:
`testWhatAreYouTesting_whenConditionForTheTest_shouldExpectedBehavior`

Reply:
Please apply to the rest